msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,57 @@
+
+  :::image type="content" source="images/actions-in-m365-app.gif" alt-text="Graphic shows the user right-clicks on the excel file and select the related suppliers.":::
+
+## Understand how Actions work
+
+Actions are the combination of intent, object, and handler. Actions represent the task that the user wants to perform where intent is the user’s desired action, object is the function to be executed, and handler is the way to perform the action on the object.
+
+The following table lists the required elements and support information for Actions:
+
+| &nbsp; | Name | Description | What is supported in this preview  
+| --- | --- | --- | ---|
+| &nbsp; | Intent | Intent is the objective a user wants to perform or achieve, such as `Open` and  `addTo`. Microsoft 365 uses intent to display Actions in locations that align with the user’s needs and intentions. Intent determines the placement, grouping, and ordering of Actions. | You can create an intent for `Open`, `addTo`, and `custom` Actions. You can use `custom` Action to create tailored action. |
+| &nbsp; | Object  | Object is the file on which the user wants to perform an action. | Currently, Actions can be triggered on content objects (files) that have an extension, such as Word, PowerPoint, Excel, PDF, and images. The files must be available in OneDrive or SharePoint and are accessible through Microsoft Graph. |
+| &nbsp; | Handlers | A handler is how the Action performs the user’s intent on the selected object. It provides the logic and functionality of the Action, creating a smooth and meaningful user experience. | Actions support `openPage` handler. The `openPage` handler allows you to directly guide users to your app's personal tab. By utilizing the `openPage` handler, you can effectively drive users to your app's dedicated pages, providing them with a rich and expansive interface to accomplish their goals. |
+
+## Design guidelines
+
+Actions enable your app to integrate seamlessly into the user's workflow and guide users to your app based on their intent and the content at hand, making task completion more efficient.
+
+:::image type="content" source="images/app-icon-context-menu.png" alt-text="The screenshot shows the app icon in context menu.":::
+
+:::image type="content" source="images/icon-slot.png" alt-text="The screenshot shows the icon slot in context menu.":::
+
+> [!NOTE]
+> The placement of Actions is determined by the Microsoft 365 platform.
+There is no guarantee that intents are grouped, and custom intents are not grouped into Actions. We are planning to introduce additional features and experiences to assist users in quickly locating the most relevant and useful actions.
```

Comment:
We don't need to mention details about grouping and ordering. As long as we say the placement is determined by Microsoft 365 platform is good enough.